sns/ut/isal_ub.c

```diff
@@ -0,0 +1,407 @@
+/*
+ * Copyright (c) 2011-2020 Seagate Technology LLC and/or its Affiliates
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ * For any questions about this software or licensing,
+ * please email opensource@seagate.com or cortx-questions@seagate.com.
+ *
+ */
+
+
+#include <stdio.h>
+#include <stdlib.h>
+#include <string.h>
+#include <getopt.h>
+#include <isa-l.h>
+
+#include "lib/ub.h"
+#include "lib/misc.h"
+#include "lib/memory.h"
+#include "lib/buf.h"
+#include "lib/time.h"
+#include "lib/arith.h"
+
+#define UB_ITER 100
+#define MMAX    255
+#define KMAX    255
+
+m0_time_t seed = 0;
+
+static int ub_init(const char *opts M0_UNUSED)
+{
+    seed = m0_time_now();
+
+    return 0;
+}
+
+static void unit_spoil( struct m0_buf *failed,
+                        const uint32_t data_count )
```

Comment:
And no space before the closing parenthesis.

Reply:
Done